eventmesh-server-go/runtime/core/protocol/grpc/heartbeat/heartbeat_service.go

```diff
@@ -51,7 +52,7 @@ func (h *HeartbeatService) Heartbeat(ctx context.Context, hb *pb.Heartbeat) (*pb
 		err     error
 	)
 	h.pool.Submit(func() {
-		resp, err = ProcessHeartbeat(h.gctx, hb)
+		resp, err = NewProcessor().Heartbeat(h.consumerMgr, hb)
```

Comment:
suggest use global variable , avoid new object action for each call Submit method.